app/models/commission_history.rb

```diff
@@ -8,13 +8,16 @@
 #  commission_before :float(24)
 #  epoch             :integer
 #  epoch_completion  :float(24)
+#  network           :string(191)
 #  created_at        :datetime         not null
 #  updated_at        :datetime         not null
 #  validator_id      :bigint           not null
 #
 # Indexes
 #
-#  index_commission_histories_on_validator_id  (validator_id)
+#  index_commission_histories_on_network                   (network)
+#  index_commission_histories_on_network_and_validator_id  (network,validator_id)
+#  index_commission_histories_on_validator_id              (validator_id)
```

Comment:
The three indexes above don't make sense when taken together. You'll never need to run a query on network + validator_id. Normally, you will have an index on [network, account] and then another index on validator_id.